### PR TITLE
update quick start with k8s.

### PR DESCRIPTION
**Pull request by morvencao**

### CODE REVIEW

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 1]
```diff
@@ -30,11 +30,10 @@ To install and configure Istio in a Kubernetes cluster, follow these instruction
 
 ## Installation steps
 
-1. Install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed in Kubernetes API server:
+1. Install all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kube api-server:
```
Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start/index.md]: all the Istio

Comment (geeknoid) [reply content/docs/setup/kubernetes/quick-start/index.md]: I'd put the "or CRDs for short" outside the link and in parenthesis.

Comment (rcaballeromx) [reply content/docs/setup/kubernetes/quick-start/index.md]: I agree:
```suggestion
1. Install all the Istio [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kubernetes API-server:
```

Comment (morvencao) [reply content/docs/setup/kubernetes/quick-start/index.md]: Done

**Comment by andraxylia** [comment]
Can we get these changes in before people start testing 1.1? We already got reports.

**Comment by rcaballeromx** [comment]
@andraxylia Yes, you can point users to the preliminary version of the site for the updated version.
/lgtm

**Pull request merged**